docs/src/model_setup/output_writers.md

```diff
@@ -53,15 +53,15 @@ Saving the u velocity field and temperature fields, the full 3D fields and surfa
 to separate NetCDF files:
 
 ```jldoctest netcdf1
-using Oceananigans, Oceananigans.OutputWriters
+using Oceananigans
 
 grid = RectilinearGrid(size=(16, 16, 16), extent=(1, 1, 1));
 
-model = NonhydrostaticModel(grid=grid);
+model = NonhydrostaticModel(grid=grid,);
```

Comment:
```suggestion
model = NonhydrostaticModel(grid=grid, tracers=(:c));
```

Let's do this so that the demonstration about the fields below does include a tracer example!

Reply:
Agreed. For some reason I'm not able to commit your suggestion directly though. I don't see any "commit" button. Weird...